systems/controllers/constrained_lqr_controller.h

```diff
@@ -0,0 +1,121 @@
+#pragma once
+
+#include "attic/multibody/contact_toolkit.h"
+#include "drake/systems/controllers/linear_quadratic_regulator.h"
+#include "drake/systems/framework/leaf_system.h"
+#include "systems/framework/output_vector.h"
+
+namespace dairlib {
+namespace systems {
+
+/*
+ * ConstrainedLQRController class that implements an LQR controller that also
```

Comment:
Somewhere in this description would be a good place to reference the appropriate publication.